sorc/emcsfc_snow2mdl.fd/snow2mdl.f

```diff
@@ -811,37 +800,27 @@ subroutine interp
 
  end subroutine interp
 
+!>  subprogram documentation block
+!!
+!! subprogram:   write_grib2
```

Comment:
My own laptop is back so I can read these comments and address.

I have told various people asking for my help that this is more important and it will NEVER EVER get done if it is not addressed and it is therefore my most important project and we're in this mess because of previous shortcuts and postponements for "urgencies".   I think we're on the same page there.  I am also disappointed in the slow progress but
with improvements in my comprehension of the process I should progress much more rapidly.

Reply:
!!subprogram write_grib2 doc removed.  !!Subprogram documentation block removed